### PR TITLE
aisetnextbuf.c OK (code_80194790.c)

### PR DESCRIPTION
**Pull request by engineer124**
This libultra audio file normally contains 1 function, but MM added a few unused functions to this file. I don't even know where to begin with documentation of these unused function, I have no clue what their intention is. If anyone has any ideas what these functions could be doing, I'm all ears.

### CODE REVIEW

File: src/code/audio/aisetnextbuf.c [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+s16* D_801D6188 = D_801D6098[0];
+s8* D_801D618C = D_801D6028[0];
+
+// Unused
+void func_80194804(s32 index) {
+    D_801D6188 = D_801D6098[index];
+    D_801D618C = D_801D6028[index];
+}
+
+s16 func_80194840(s32 arg0) {
```
Comment (EllipticEllipsis) [inline src/code/audio/aisetnextbuf.c:63]: You could note this is only used below.

File: src/code/audio/aisetnextbuf.c [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    }
+}
+
+// Unused
+s32 func_801948B0(s32* arg0, s32* arg1) {
+    s32 temp_v0;
+    s32 phi_s0 = *arg0;
+    s32 phi_s1 = *arg1;
+
+    while (true) {
```
Comment (EllipticEllipsis) [inline src/code/audio/aisetnextbuf.c:88]: I assume you couldn't get this to be a for loop?

Comment (engineer124) [reply src/code/audio/aisetnextbuf.c:88]: You could do:
```
phi_s0 = *arg0;
for (phi_s1 = *arg1;; phi_s1 -= temp_v0, phi_s0++) {
```
You can't have both initializers inside the for-loop, and trying to move `phi_s1 < temp_v0` into a for-loop breaks everything

Comment (engineer124) [reply src/code/audio/aisetnextbuf.c:89]: The issue with `phi_s1 < temp_v0` is that the limit `temp_v0` is recalculated every loop, so I think that's what's breaking regalloc.

Comment (EllipticEllipsis) [reply src/code/audio/aisetnextbuf.c:89]: Mm, fair enough, sounds like it's wouldn't really make sense that way anyway.

**Review by EllipticEllipsis — APPROVED** [review]
The only thing we could do is give these a file-specific prefix, I suppose, but since they're unused there's not a lot of point anyway.

**Review by hensldm — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**